src/frontends/pytorch/src/op/erfc.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/erf.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_erfc(const NodeContext& context) {
+    // aten::erf(Tensor self) -> Tensor
+    // aten::erf.out(Tensor self, Tensor(!a) out) -> Tensor(!a)
+    num_inputs_check(context, 1, 2);
+    auto x = context.get_input(0);
+    auto xdtype = x.get_element_type();
+
+    // in torch, Erfc return always float dtype, while ov cast to input dtype
+    if (xdtype.is_dynamic() || !xdtype.is_real()) {
+        x = context.mark_node(std::make_shared<v0::Convert>(x, element::f32));
+    }
+
+    // apply Erf to the input tensor 'x'
+    auto y = context.mark_node(std::make_shared<v0::Erf>(x));
+
+    // create 'ones' to use to calculate complementary of Erf output
+    auto ones = context.mark_node(make_shared<v0::Constant>(element::f32, Shape{}, 1.0f))->output(0);
+    ones = context.mark_node(make_shared<v1::ConvertLike>(ones, y));
```

Comment:
```suggestion
    auto ones = context.mark_node(make_shared<v0::Constant>(element::f32, Shape{}, 1.0f))->output(0);
    align_eltwise_input_types(context, x, ones);
    
    // apply Erf to the input tensor 'x'
    auto y = context.mark_node(std::make_shared<v0::Erf>(x));
```
Your approach will not work for dynamic case. `align_eltwise_input_types` will align type of FP32 `ones` input and `x` the result will be equal type, which will work for FP64, INT and dynamic case.